.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+C++:
+  - cpp/**/*
+  - change-notes/**/*cpp*
+
+C#:
+  - csharp/**/*
+  - change-notes/**/*csharp*
+
+Java:
+  - java/**/*
+  - change-notes/**/*java.*
+
+JS:
+  - javascript/**/*
+  - change-notes/**/*javascript*
+
+Python:
+  - python/**/*
+  - change-notes/**/*python*
+
+documentation:
+  - **/*.qhelp
+  - **/*.md
```

Comment:
Sounds good to me, but please also include: `docs/***/*` in this category.

Reply:
Done! Thanks for the suggestion.

Reply:
Hinaroo$6798